### PR TITLE
Added response to transfer

### PR DESCRIPTION
**Pull request by akoidan**

### CODE REVIEW

File: src/main/java/com/icthh/xm/ms/balance/service/BalanceService.java [hunk 1]
```diff
@@ -232,6 +233,11 @@ public void transfer(TransferBalanceRequest transferRequest) {
         metricService.updateMaxMetric(targetBalance);
         balanceChangeEventRepository.save(eventFrom);
         balanceChangeEventRepository.save(eventTo);
+        return TransferDTO
```
Comment (sergeysenja1992) [inline src/main/java/com/icthh/xm/ms/balance/service/BalanceService.java]: any test for it?

File: src/main/java/com/icthh/xm/ms/balance/web/rest/BalanceResource.java [hunk 2]
```diff
@@ -159,8 +160,9 @@ public BalanceResource(BalanceService balanceService, BalanceQueryService balanc
     @PreAuthorize("hasPermission({'transferRequest': #transferRequest}, 'BALANCE.TRANSFER')")
     @PostMapping("/balances/transfer")
     @Timed
-    public void transferBalance(@Valid @RequestBody TransferBalanceRequest transferRequest) {
-        balanceService.transfer(transferRequest);
+    public ResponseEntity<TransferDTO> transferBalance(@Valid @RequestBody TransferBalanceRequest transferRequest) {
+        TransferDTO response = balanceService.transfer(transferRequest);
+        return ResponseUtil.wrapOrNotFound(Optional.ofNullable(response));
```
Comment (sergeysenja1992) [inline src/main/java/com/icthh/xm/ms/balance/web/rest/BalanceResource.java:165]: please, add test

File: src/main/java/com/icthh/xm/ms/balance/service/BalanceService.java [hunk 3]
```diff
@@ -210,7 +211,7 @@ public BalanceChangeEventDto charging(Balance balance, ChargingBalanceRequest ch
 
     @Transactional
     @LogicExtensionPoint("Transfer")
-    public void transfer(TransferBalanceRequest transferRequest) {
+    public TransferDTO transfer(TransferBalanceRequest transferRequest) {
```
Comment (v-kyrychenko) [inline src/main/java/com/icthh/xm/ms/balance/service/BalanceService.java]: rename TransferDTO according to check style requirements - **TransferDto**

File: src/main/java/com/icthh/xm/ms/balance/web/rest/BalanceResource.java (hunk 2 above)
Comment (v-kyrychenko) [reply src/main/java/com/icthh/xm/ms/balance/web/rest/BalanceResource.java:165]: done

**Review by ishkurko — APPROVED** [review]

**Pull request merged**